src/cuga/backend/tools_env/registry/registry/api_registry_server.py

```
@@ -554,16 +630,24 @@ async def clear_agent_cache(
     try:
         if agent_id:
             if agent_id in agent_registries:
-                mgr, _ = agent_registries.pop(agent_id)
+                mgr, reg = agent_registries.pop(agent_id)
                 await mgr.shutdown()
+                try:
+                    await reg.cleanup()
+                except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)